### PR TITLE
Member page edit and show

### PR DESCRIPTION
**Pull request by thesuss**
PT Story: https://www.pivotaltracker.com/story/show/164877984

image

image

### CODE REVIEW

File: app/views/pages/remiss.html [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-<p>N&auml;r du tar dig an en remiss &auml;r det viktigt att &aring;terkoppla till remitterande kollega.</p>
+<p>Vi &auml;r ett n&auml;tverk av yrkeskunniga f&ouml;retagare som alla &auml;r seri&ouml;sa och ser till hundens b&auml;sta. Kom ih&aring;g att vi alla gynnas av bransch&ouml;verskridande samarbeten.</p>
+
+<p>Remittera g&auml;rna till varandra genom att lotsa kunder med behov av andra tj&auml;nster vidare till en kollega inom SHF. Se till att h&aring;lla en n&auml;ra kontakt och delge varandra journalanteckningar, iakttagelser, rekommenderade behandlingar och liknande s&aring; att kunden f&aring;r maximal hj&auml;lp tack vare ert samspel. Ett snabbt telefonsamtal mellan en behandlande hundfysioterapeut och hundpsykolog kan g&ouml;ra stor skillnad f&ouml;r det enskilda fallet.</p>
+
+<p><a href="/hundforetag">hitta mailadress</a></p>
+
+<h3>Information att delge vid remittering:</h3>
+
+<ul>
```
Comment (weedySeaDragon) [inline app/views/pages/remiss.html:9]: Minor _opinion_ comment:

I think this should be a numbered list (`%ol`).  The bullets make it less clear that each item is really like a _label_ and folks should fill out info after them.
I think this is clearer:

1. Hund&auml;garens namn:
2. Hund&auml;garens kontaktdetaljer:
3. Hundnamn, ras/raser, &aring;lder, k&ouml;n:
4. Orsak till remiss:

(but with more blank lines/space after " Hund&auml;garens kontaktdetaljer:" as needed.)

Most forms have numbered items to fill out, not bullets.

**Review by weedySeaDragon — APPROVED** [review]

**Comment by patmbolger** [comment]
I can _barely_ see the edit button:

Screen Shot 2019-03-30 at 9 16 20 AM

(Chrome on Mac)

**Comment by patmbolger** [comment]
You probably should also fix this page:

Screen Shot 2019-03-30 at 9 17 51 AM

**Comment by thesuss** [comment]
If its only this @patmbolger I think it can be merged because 1) is handled by the btns PR and 2) is not put in production.

**Pull request merged**